### ISSUE TITLE
Tab completion fails with master of prompt_toolkit

### ISSUE THREAD

**Issue by takluyver** [scopatz/xonsh#657]
If I tab complete when there's more than one possible completion, I get the error below. I'm on master of prompt_toolkit, working on integrating it into IPython. If I revert to 0.57, it behaves nicely.

```
Exception in thread Thread-6:
Traceback (most recent call last):
  File "/usr/lib/python3.4/threading.py", line 920, in _bootstrap_inner
    self.run() 
  File "/usr/lib/python3.4/threading.py", line 868, in run
    self._target(*self._args, **self._kwargs)
  File "/home/takluyver/.local/lib/python3.4/site-packages/prompt_toolkit/intern
    completions = list(buffer.completer.get_completions(document, complete_even)
  File "/home/takluyver/.local/lib/python3.4/site-packages/xonsh/ptk/completer.s
    self.reserve_space()
  File "/home/takluyver/.local/lib/python3.4/site-packages/xonsh/ptk/completer.e
    window = cli.application.layout.children[1].children[1].content
AttributeError: 'FloatContainer' object has no attribute 'children'
```

**Comment by takluyver** [comment]
Sorry for the noise, I see this was already reported in #643 and fixed in #647. I'll just run *all the things* from master!

**Comment by scopatz** [comment]
These are my kind of issues :)
